### DISCUSSION TITLE
通知を受け取りませんか？のポップアップが何度も出てくる

### DISCUSSION THREAD

**Discussion by radian-jp** [increments/qiita-discussions#428] [要望・アイデア・不具合]
最近、ログイン時に通知を受け取りませんか？のポップアップが出てくるようになりました。
通知は邪魔なので基本的に受け取らないのですが、ポップアップを閉じても
ブラウザを閉じてログインし直すと、また出てくるので迷惑です。
アカウント単位で設定を保存するなりの改善をお願いしたいです。

**Comment by ohakutsu** [comment]
@radian-jp 
ご不便をおかけして申し訳ございません。

「通知を受け取りませんか？」のポップアップは、本来、閉じると一定期間の再表示がされないようにしているのですが、それが正しく動作していない可能性があります。

もしよろしかったら問題が発生している、

- ブラウザの種類
- ブラウザのバージョン
- シークレットモードの有無

を教えていただけないでしょうか :pray:

**Reply by radian-jp** [reply]
Google Chrome 109.0.5414.120（Official Build）（64 ビット）
シークレットモードは使用していません。
Cookie AutoDeleteという拡張機能を入れています。

一定期間ではなく、一度拒否したら端末変えようがずっと再表示して欲しくないです。

**Reply by ohakutsu** [reply]
@radian-jp 
ご返信いただきありがとうございます。

現在はブラウザに設定を保持しており、`Cookie AutoDelete`にて削除されてしまっているのが原因の可能性があります。
現時点では上記のようになっているため、`Cookie AutoDelete`でQiitaを除外していただけると解消されると思われます。

また、

- アカウント単位で設定を保存
- 一度拒否したら端末変えても再表示されない

については、すぐ対応はできないですが実現できないかを検討させていただきます :bow:

**Reply by tanaka-asami415** [reply]
@tsubolabo 
こちらの通知（Webプッシュ通知）は、ブラウザの設定の通知をオフにしていただく必要がございます。
以下ヘルプページに記載の手順で、設定をご変更ください🙇
https://help.qiita.com/ja/articles/qiita-push-notification#part-9ef1529433b53abf
